apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/CreateLoadJobOptions.cs

```diff
@@ -91,6 +91,14 @@ public sealed class CreateLoadJobOptions : JobCreationOptions
         /// </summary>
         public string NullMarker { get; set; }
 
+        /// <summary>
+        /// The character encoding of the data. The default value is UTF-8. BigQuery decodes the data after the
+        /// raw, binary data has been split using the values of the <see cref="Quote"/> and <see cref="FieldDelimiter"/>
+        /// properties.
+        /// Refer https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationLoad.FIELDS.encoding for more details.
```

Comment:
Please change this to "See" or "Refer to" - just "refer xyz" doesn't sound quite right.